docs/containers-storage.conf.5.md

```diff
@@ -34,6 +34,11 @@ The `storage` table supports the following options:
   container storage graph dir (default: "/var/lib/containers/storage")
   Default directory to store all writable content created by container storage programs.
 
+  The rootless graphroot path supports three substations:
```

Comment:
Maybe it's me pre-tea, but I'm not sure what a "substation" is in this context.  Can you define or provide an example?

Reply:
I'm guessing `substitutions`.

Reply:
LOL, I guess I shouldn't copy-paste without reading properly. Will fix